config/initializers/slack.rb

```diff
@@ -0,0 +1,3 @@
+Slack.configure do |config|
+  config.token = ENV['SLACK_API_TOKEN']
```

Comment:
Perhaps `ENV.fetch('SLACK_API_TOKEN')`, so that the app immediately crashes on boot if this isn't set?

Reply:
💕